### DISCUSSION TITLE
Training Log 2023-03-03

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#373] [Training Logs 训练日志]
# CPM-Live Training Log (March, 3)

Time: March, 3 2023 19:00

Recorder: @zh-zheng

## Loss
- Begin: 2.11
- End: 1.86
	
## Completed Data
- $\approx$ 2664.01GB

## Average Grad Norm
- 2.63

## Progress
- 97.06%

## Comment

The loss is substantially reduced! Probably because the newly added training data is easier to learn.
